src/azul/indexer.py

```diff
@@ -183,3 +183,72 @@ def merge(cur_bundles, new_bundles):
         for bundle in cur_bundles_by_id.values():
             assert bundles.setdefault(bundle['uuid'], bundle) is bundle
         return list(bundles.values())
+
+    def delete(self, dss_notification: Mapping[str, Any]) -> None:
+        bundle_uuid = dss_notification['match']['bundle_uuid']
+        bundle_version = dss_notification['match']['bundle_version']
+        index = dss_notification['match']['index']
+
+        es_client = self.properties.elastic_search_client
+
+        search_query = {
+            "query": {
+                "match": {
+                    "bundles.uuid.keyword": bundle_uuid
+                }
+            }
+        }
+
+        current_doc = es_client.search(index=index,
```

Comment:
This variable name is misleading. The search() method returns a response with potentially multiple documents.